config/database.js

```diff
@@ -1,4 +1,4 @@
-const UserRights = require("./userRights.js");
+const UserRights = require("../public/dist/js/userRights.js");
```

Comment:
i'm not an expert in js but hardcoding the path for the require seems a bit strange.

Reply:
I suppose 'public/dist/js' could be stored in some global constant. Apart from that, I am not aware of any good ways to avoid this hardcoding in JavaScript.

Most opensource projects that I have seen also have hardcoded strings, for example: [Express](https://github.com/expressjs/express/blob/master/lib/application.js).

So far I have not found it to be a nuisance as any modern IDE has a renaming/moving function which automatically finds all the occurrences of a file and changes all the strings appropriately.

Reply:
You're right @Daanra . This is how js imports work and there's not much to do about it. The part that was icky was the fact that we're in a .js file and we're providing a path to public/dist/js... i would have expected that required knows the base path where to look for js files. but then again, i don't know myself a better way :)